client/my-sites/sidebar-unified/use-site-menu-items.js

```diff
@@ -32,12 +33,31 @@ const useSiteMenuItems = () => {
 	 * avoids a situation where the user might be left with an
 	 * empty menu.
 	 */
-	return (
-		menuItems ??
-		buildFallbackResponse( {
-			siteDomain,
-		} )
+	const shouldShowLinks = true;
+	const shouldShowTestimonials = useSelector( ( state ) =>
+		get( state.siteSettings.items, [ selectedSiteId, 'jetpack_testimonial' ], false )
 	);
+	const shouldShowPortfolio = useSelector( ( state ) =>
+		get( state.siteSettings.items, [ selectedSiteId, 'jetpack_portfolio' ], false )
+	);
+	const shouldShowWooCommerce = true;
+	const shouldShowApperanceHeaderAndBackground = true;
+	const shouldShowAdControl = false;
+	const shouldShowAMP = false;
+	const shouldShowThemeOptions = true;
+	const fallbackOptions = {
+		siteDomain,
+		shouldShowLinks,
+		shouldShowTestimonials,
+		shouldShowPortfolio,
+		shouldShowWooCommerce,
+		shouldShowApperanceHeaderAndBackground,
+		shouldShowAdControl,
+		shouldShowAMP,
+		shouldShowThemeOptions,
+	};
```

Comment:
It feels a little odd to me that we have this component decide which elements to show. Any reason `buildFallbackResponse()` couldn't figure that out itself?

Reply:
We require hooks to look inside of global state + `buildFallbackResponse()` is called conditionally + hooks can't be called conditionally.